custom/panel_templates/Default/assets/css/custom.css

```diff
@@ -351,6 +351,12 @@ body {
     .sidebar {
         width: 17rem !important;
     }
+    
+    .sidebar.toggled {
+        width: 7rem !important;
```

Comment:
This seems to force the sidebar to be always open on mobile (it should be wrapped in `@media (min-width: 768px)` to only apply to bigger screens)

Reply:
it is in media query

Reply:
Thanks, this is me being lazy and not checking the above lines